node/src/main/scala/pravda/node/servers/ApiRoute.scala

```diff
@@ -75,12 +76,13 @@ class ApiRoute(abciClient: AbciClient, db: DB, abci: Abci)(implicit executionCon
   val longUnmarshaller: Unmarshaller[String, Long] =
     Unmarshaller.strict(s => s.toLong)
 
-  val balances: Entry[Address, NativeCoin] = balanceEntry(db)
-  val eventsByAddress = eventsByAddressEntry(db)
-  val events = eventsEntry(db)
-  val txIdIndex = txIdIndexEntry(db)
-  val transferEffectsByAddress = transferEffectsEntry(db)
-  val transactionsByAddress = transactionsEntry(db)
+  val balances: Entry[Address, NativeCoin] = balanceEntry(applicationStateDb)
+  val events = eventsEntry(applicationStateDb)
+  val txIdIndex = txIdIndexEntry(applicationStateDb)
```

Comment:
Are you sure that transaction index should be stored in application state?

Reply:
You are right. Is not necessary to store txIdIndex in applicationStateDb
I will change it